### PR TITLE
CLDR-10963 Seed data for Aragonese (an)

### PR DESCRIPTION
**Pull request by JCEmmons**
[CLDR-10963] 

##### Checklist

- [ ] Issue filed: https://unicode-org.atlassian.net/browse/CLDR-10963
- [ ] Updated PR title and link in previous line to include Issue number

[CLDR-10963]: https://unicode-org.atlassian.net/browse/CLDR-10963

### CODE REVIEW

**Comment by JCEmmons** [comment]
I have a contributor who is ready willing and able, so wanted to get this in.

File: tools/java/org/unicode/cldr/util/data/country_language_population_raw.txt [hunk 1]
```diff
@@ -1154,6 +1154,7 @@ South Sudan	SS	"10,204,581"	27%	"20,010,000,000"		Arabic	ar	27%			https://www.ci
 South Sudan	SS	"10,204,581"	27%	"20,010,000,000"	official	English	en	27%			https://www.cia.gov/library/publications/the-world-factbook/geos/od.html http://en.wikipedia.org/wiki/South_Sudan
 South Sudan	SS	"10,204,581"	27%	"20,010,000,000"		Nuer	nus	5.6%			http://www.ethnologue.com/language/nus unknown literacy
 Spain	ES	"49,331,076"	98%	"1,778,000,000,000"	official_regional	Asturian	ast	"656,000"			http://www.ethnologue.com/show_language.asp?code=ast
+Spain	ES	"49,331,076"	98%	"1,778,000,000,000"		Aragonese	an	"25,500"
```
Comment (srl295) [inline tools/java/org/unicode/cldr/util/data/country_language_population_raw.txt]: :+1: this was my first question on seeing the likely subtags… 

 but did you update The Spreadsheet™ also?

Comment (JCEmmons) [reply tools/java/org/unicode/cldr/util/data/country_language_population_raw.txt]: This is "the spreadsheet", and it's the only thing that really matters.  likely subtags is generated from this.

Comment (srl295) [reply tools/java/org/unicode/cldr/util/data/country_language_population_raw.txt]: Carry on, then!

**Pull request merged**